code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -88,6 +88,7 @@
 	poison_per_bite = 3
 	var/atom/movable/cocoon_target
 	var/fed = 0
+	var/static/list/consumed_mobs = list() //mobs that have been consumed by nurse spiders to lay eggs

```

Comment:
This list needs to be `Cut` when the spider is destroyed; likewise, any mob that is destroyed needs to be removed from this list, or else it will fail to GC. 

I realize that mobs aren't soft deleted, but this is still good practice.